src/main/java/org/jitsi/jigasi/JvbConference.java

```diff
@@ -150,6 +153,12 @@ public class JvbConference
      */
     private String meetingId;
 
+    /**
+     * A thread pool used to offload xmpp execution in a new thread to avoid blocking
+     * xmpp threads.
+     */
+    public static final ExecutorService xmppExecutorPool = Util.createNewThreadPool("xmpp-executor-pool");
```

Comment:
Do we not have a generic "io" thread pool in jigasi?

Reply:
Nothing, this becomes one for the xmpp handling.